controllers/datadogagent/feature/utils/utils.go

```diff
@@ -19,6 +19,7 @@ import (
 // Process Checks utils
 
 const RunInCoreAgentMinVersion = "7.53.0-0"
+const LanguageDetectionRunInCoreAgentMinVersion = "7.57.0-0"
```

Comment:
I'm considering collapsing these into one version we check and continuously update until we GA. This would greatly simplify the code.